From b0b86fde5e5aa939f16155b9a4bcae4d4e8cae50 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 21 Feb 2025 14:46:35 +0100
Subject: [PATCH 1624/1695] drivers: stm32mp1_stpmic1: remove unused includes

Drop the includes not anymore in use and add 'interrupt.h' that
was previously included through 'stm32mp1_pwr.h'.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git fceaa13ff5a9705b5d20fc60925339b0c48529ae drivers: stm32mp1_stpmic1: remove unused includes]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I6136fcd70f9626d075f724697687f950b0fe0229
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/441055
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c
index 0e8d7b035..d1a5070ea 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c
@@ -8,15 +8,14 @@
 #include <drivers/regulator.h>
 #include <drivers/stm32_i2c.h>
 #include <drivers/stm32mp1_stpmic1.h>
-#include <drivers/stm32mp1_pwr.h>
 #include <drivers/stpmic1.h>
 #include <drivers/stpmic1_regulator.h>
 #include <dt-bindings/mfd/st,stpmic1.h>
 #include <io.h>
 #include <keep.h>
 #include <kernel/boot.h>
-#include <kernel/delay.h>
 #include <kernel/dt.h>
+#include <kernel/interrupt.h>
 #include <kernel/notif.h>
 #include <kernel/panic.h>
 #include <kernel/pm.h>
-- 
2.47.2

